cmake/Modules/FindCaffe.cmake

```diff
@@ -0,0 +1,21 @@
+# Caffe package
+unset(Caffe_FOUND)
+
+set(Caffe_DIR /home/kunal/ssd/caffe/distribute)
```

Comment:
Can you make this line a comment, and explain that the user can set Caffe_DIR to the path to the version of Caffe that they would like to use?  It seems odd to include your caffe directory in a public commit.